src/shared/store/actions/utils/makeESQuery.ts

```diff
@@ -56,8 +56,8 @@ export const makeESQuery = (query?: List['query']) => {
     }
     if (query.textQuery) {
       must.push({
-        query_string: {
-          query: `${query.textQuery}~`,
+        match: {
```

Comment:
Could you add a comment to explain why you are using `match` here?

Reply:
I added it. It's from the implementation of the full_text_search query param on KG. https://github.com/BlueBrain/nexus-kg/pull/675/files#diff-4aa2f8ee4ce4694213bc1e9add6895c0R9

Reply:
Not sure the comment is 100% needed in the code itself but if you decide to keep it there, can you link to the actual file rather that the PR: https://github.com/BlueBrain/nexus-kg/blob/master/src/test/resources/search/query-schema-rev-createdBy-q.json#L10-L12